### PR TITLE
Added Cluedo

### PR DESCRIPTION
**Pull request by marcelherd**
Submitting this PR means you have:

- [x] searched the [readme](https://github.com/edm00se/awesome-board-games/blob/master/readme.md) file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [x] and will follow [the desired format](https://github.com/edm00se/awesome-board-games/blob/master/formatting.md)
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build on Travis CI. This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines](https://github.com/edm00se/awesome-board-games/blob/master/contributing.md) or [open an issue](https://github.com/edm00se/awesome-board-games/issues/new).

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -134,7 +134,7 @@
 
 > The great 7 Wonders concepts adapted especially for two players. The feel is very '7 Wonders' with a few refinements which many will with made it to the main game such as the reworked military and science victories as well as new commerce dynamics. Playing a game always leaves me wanting to play again.
 
-![7_Wonders_Duel_game_image](https://cf.geekdo-images.com/small/img/1fkPb9DgZ8n_F99jAhJpKgShYPY=/fit-in/200x150/pic2576399.jpg)
+![7_Wonders_Duel_game_image](https://cf.geekdo-images.com/small/img/1fkPb9DgZ8n_F99jAhJpKgShYPY=/fit/200x150/pic2576399.jpg)
```
Comment (edm00se) [inline readme.md]: This line is unrelated to your change, it should not have been touched. Please revert it.

```suggestion
![7_Wonders_Duel_game_image](https://cf.geekdo-images.com/small/img/1fkPb9DgZ8n_F99jAhJpKgShYPY=/fit-in/200x150/pic2576399.jpg)
```

**Comment by marcelherd** [comment]
@edm00se How do I keep the commit out of this PR? It was added automatically

**Comment by edm00se** [comment]
Normally the git operation would look like:
- create new branch with the added changes
- reset the first branch to before that last change

Since this gets complicated due to two two sets of changes independently, I would do the following:
- create a new branch from the commit before your first change, this will be for Sheriff of Nottingham
- cherry-pick the commit that includes the Sheriff of Nottingham change into that new branch
- reset the first branch, the one this PR seeks to pull in, to before the Sheriff of Nottingham, to keep it just for the Cluedo changes

**Comment by marcelherd** [comment]
I will create a new fork and work in branches

**Comment by edm00se** [comment]
That works as well.
